src/db/models/user.js

```diff
@@ -0,0 +1,17 @@
+import mongoose from 'mongoose';
+import passportLocalMongoose from 'passport-local-mongoose';
+
+const Schema = mongoose.Schema;
+
+const User = new Schema({
+  name: { type: String, required: true },
```

Comment:
I think it reads better with columns separated by newline regardless of properties of said column.